.github/eslint.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  parserOptions: {
+    ecmaVersion: 2023,
+  },
+  env: {
+    node: true,
+  },
+  extends: ["eslint:recommended"],
```

Comment:
Single quotes are used by ~80% of the JS community, also it will reduce noise in this PR.
```suggestion
  extends: ["eslint:recommended"],
  rules: {
    quotes: [
      'error',
      'single'
    ]
  }
```